app/com/arpnetworking/database/h2/triggers/BaseUpdateEtagTrigger.java

```diff
@@ -32,10 +33,13 @@
     /**
      * Public constructor.
      *
-     * @param sequenceName Name of the sequence that should be updated.
+     * @param updateTableName Name of the table that should be updated.
+     * @param orgIndex The index to find the organization for which to update the etag.
      */
-    public BaseUpdateEtagTrigger(final String sequenceName) {
-        _queryStatement = String.format("SELECT NEXTVAL('%s');", sequenceName);
+    public BaseUpdateEtagTrigger(final String updateTableName, final int orgIndex) {
+        _updateStatement = String.format("UPDATE %s SET etag = etag + 1 where organization = ?", updateTableName);
```

Comment:
So is it not possible to model this with EBean? -- it wasn't before because we were using a counter.